lib/IRGen/GenProto.cpp

```diff
@@ -961,11 +967,13 @@ static bool isDependentConformance(
     if (assocConformance.isInvalid())
       return false;
 
+    auto rootConformance = assocConformance.getConcrete()->getRootConformance();
```

Comment:
I think you need to guard this by `assocConformance.isAbstract()`

Reply:
... `! assocConformance.isAbstract()`

Reply:
Yes, I figured out as much after doing a local build.